contrib/implements/internal/implements/cast.go

```diff
@@ -64,35 +67,40 @@ func stubCFunctions(libname string) (CFunctions, error) {
 	if cstub == "" {
 		return nil, fmt.Errorf("no C stub available for '%s'", libname)
 	}
-	var conf cc.Config
-	conf.PreprocessOnly = true
-	conf.IgnoreInclude = regexp.MustCompile(`^<.+>$`)
+	conf, err := cc.NewConfig(runtime.GOOS, runtime.GOARCH, "-D_FILE_OFFSET_BITS=64")
+	if err != nil {
+		return nil, err
+	}
+
 	src := []cc.Source{
-		{Name: "typestubs", Value: typeStubs, DoNotCache: true},
-		{Name: libname, Value: cstub, DoNotCache: true},
+		{Name: "<predefined>", Value: conf.Predefined},
+		{Name: "<builtin>", Value: cc.Builtin},
+		{Name: "typestubs", Value: typeStubs},
+		{Name: libname, Value: cstub},
 	}
-	inc := []string{"@", "/usr/local/include", "/usr/include"}
-	cAST, err := cc.Parse(&conf, inc, nil, src)
+	cAST, err := cc.Parse(conf, src)
 	if err != nil {
 		return nil, err
 	}
-	cfMap := map[cc.StringID]CFunction{}
-	deprecated := cc.String("deprecated")
-	for i := range cAST.Scope {
-		for _, n := range cAST.Scope[i] {
+	cfMap := map[cc.StringValue]CFunction{}
```

Comment:
I see that [cc.StringValue](https://pkg.go.dev/modernc.org/cc/v4#StringValue) is basically a type definition for string. But this should be OK.